image_explorer/templates/html/image_explorer_edit.html

```diff
@@ -10,6 +10,13 @@
           </div>
           <span class="tip setting-help">{% trans "The title of the Image Explorer that is displayed to the user" %}</span>
       </li>
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+            <input type="checkbox" class="setting-input edit-hot-spots" id="edit_hot_spots" {% if self.hot_spots %}checked="checked"{% endif %}>
+            <label class="label setting-label label-block" for="edit_hot_spots">Image explore calculated by percentage</label>
+          </div>
+          <span class="tip setting-help">{% trans "If set to true, calculates by percentage <br /> If set to false, calculates by pixels" %}</span>
```

Comment:
I don't think the text is correct. You can still use either percentage or pixels in the specification. The only difference is in how the given location is interpreted. When the new flag is enabled, it is interpreted as the center of the hotspot, without it, it is assumed to point to the top left corner of the hotspot image.